comments-by-guideline-and-success-criterion.md

```diff
@@ -239,8 +240,8 @@ See also the [Comments on Closed Functionality](#comments-on-closed-functionalit
 <ul><li>Situations where it is unclear whether the exception of a two-dimensional layout might apply.</li>
 <li>Situations where 1.4.10 Reflow cannot be met.</li>
 <li>Are there contexts where text sizing might be an acceptable alternative to 1.4.10 Reflow?</li>
-<li>Are there situations where the content technology and/or platform software do not support reflow? If there are, please explain and indicate whether applications built for this environment should fail this success criterion or have an exception so that this criterion doesn't get applied.</li>
-<li>Are there situations where the content technology and/or platform software prevents users from resizing or zooming text, changing the viewport and screen orientation (all cases where reflow of content would typically be needed)? If there are, please explain and indicate whether applications built for this environment should fail this success criterion or have an exception so that this criterion is not applied.</li></ul></div>
+<li>Are there situations where the content technology and/or platform software do not support reflow? If there are, please explain and indicate whether applications built for this environment should fail this success criterion or have an exception.</li>
```

Comment:
All of these things are inside of an editor's note that we have changed in another PR. Not necessary to apply these changes and I'll resolve them when handling merge conflicts.